app/mailers/views/application_mailer/feedback_email.html.haml

```diff
@@ -1,6 +1,7 @@
+.header_image
+  =image_tag URI.join(root_url, "assets/mailer/feedback.png"), size: '494x263'
 = t('.content', email: @email).html_safe
 %br
 %br
-
-= @text
-%br
+.feedback_text
+  %q= @text
```

Comment:
Is this text a quotation? Why is it tagged <q>?

Reply:
Because i wanna let the text between quotations